pkg/northbound/gnmi/v2/get_test.go

```diff
@@ -31,16 +31,18 @@ import (
 	"testing"
 )
 
-func createServer(t *testing.T) (*Server, *gomock.Controller, *atomixtest.Test) {
+func createServer(t *testing.T) (*Server, *gomock.Controller, *atomixtest.Test, *gnmitest.MockStore, *gnmitest.MockPluginRegistry) {
 	mctl := gomock.NewController(t)
 	test, cfgStore, txStore := testStores(t)
+	registryMock := gnmitest.NewMockPluginRegistry(mctl)
+	topoMock := gnmitest.NewMockStore(mctl)
 	return &Server{
 		mu:             sync.RWMutex{},
-		pluginRegistry: gnmitest.NewMockPluginRegistry(mctl),
-		topo:           gnmitest.NewMockStore(mctl),
+		pluginRegistry: registryMock,
+		topo:           topoMock,
 		transactions:   txStore,
 		configurations: cfgStore,
-	}, mctl, test
+	}, mctl, test, topoMock, registryMock
```

Comment:
I would suggest to define an struct and include all these mocks and information that should be returned to the caller.

Reply:
Yeah, I was already working towards that in the current PR.